rebasehelper/specfile.py

```diff
@@ -568,297 +521,7 @@ def set_extra_version(self, extra_version: Optional[str], version_changed: bool)
         self.set_release(release)
         # TODO: in some cases it might be necessary to modify Source0
 
-    @saves
-    def set_tag(self, tag: str, value: str, preserve_macros: bool = False) -> None:
```

Comment:
So happy that this beast is gone :D

Reply:
Same here :slightly_smiling_face: The new implementation in specfile is way better IMHO, though not 100%.